### ISSUE TITLE
Cannot read property 'ansi' of undefined

### ISSUE THREAD

**Issue by enjikaka** [chalk/ansi-styles#18]
Running example from https://github.com/chalk/ansi-styles

spits out this error in console:

```
Hello world!
/Users/jeremy/Dev/ascii/ansi.js:10
console.log(style.bgColor.ansi.hsl(120, 80, 72) + 'Hello world!' + style.bgColor.close);
                         ^

TypeError: Cannot read property 'ansi' of undefined
    at Object.<anonymous> (/Users/jeremy/Dev/ascii/ansi.js:10:26)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:138:18)
    at node.js:974:3
```

Issue in Ansy: https://github.com/IonicaBizau/ansy/issues/1#issuecomment-203933927

cc @IonicaBizau

**Comment by Qix-** [comment]
Hi! See #15 - specifically, [the post mortem](https://github.com/chalk/ansi-styles/issues/15#issuecomment-202589714). We had a publishing snafu a few days ago and we realized a minor version was released with some broken code and had to roll back.

Your library depends on the new breaking changes (which is great! It's a much better API). Until we get a major release out, `ansy` will need to change the dependency from `ansi-styles@2.2.0` to `chalk/ansi-styles` temporarily.

Once a major release is out, the new version will be `ansi-styles@3.0.0`. This will be happening in the coming days.

Really sorry for the inconvenience here. This is ongoing and has been as much of a pain for us as it is for downstream maintainers. :heart:
